test/users.test.js

```diff
@@ -182,6 +208,84 @@ test('Create a new user', function(done) {
         });
 });
 
+test('Get All Users', function(done) {
+    expect.assertions(4);
+    request(
+        {
+            method: 'POST',
+            baseUrl: 'http://127.0.0.1:' + config.port,
+            uri: '/user/getAllOf',
+            json: true,
+            body: {
+                eaeUsername: adminUsername,
+                eaeUserToken: adminPassword,
+                userType: 'all'
+            }
+        },
+        function(error, response, body) {
+            if (error) {
+                done.fail(error.toString());
+            }
+            expect(response).toBeDefined();
+            expect(response.statusCode).toEqual(200);
+            expect(body).toBeDefined();
+            expect(body).toEqual([{username: 'adminUsers'},{username: 'NotLegit'}]);
+            done();
+        });
+});
+
+test('Get All Admin Users', function(done) {
+    expect.assertions(4);
+    request(
+        {
+            method: 'POST',
+            baseUrl: 'http://127.0.0.1:' + config.port,
+            uri: '/user/getAllOf',
+            json: true,
+            body: {
+                eaeUsername: adminUsername,
+                eaeUserToken: adminPassword,
+                userType: 'admin'
```

Comment:
try to be consistent in the naming convention. you use "STANDARD" above then "all" and "admin" in lower cases

Reply:
testing the conversion from lower case to upper in the function

Reply:
then write it in the description of the test because that's no obvious :)